pkg/config/app_config.go

```diff
@@ -200,12 +213,16 @@ func (c *AppConfig) GetAppState() *AppState {
 	return c.AppState
 }
 
+func (c *AppConfig) GetUserConfigFiles() []string {
+	return c.UserConfigFiles
+}
+
 func (c *AppConfig) GetUserConfigDir() string {
 	return c.UserConfigDir
 }
 
 func (c *AppConfig) ReloadUserConfig() error {
-	userConfig, err := loadUserConfigWithDefaults(c.UserConfigDir)
+	userConfig, err := loadUserConfigWithDefaults(c.UserConfigFiles)
```

Comment:
I think we should put the validation logic back inside `loadUserConfig` so that when reloading the user config we can handle any errors gracefully (e.g. for some reason a config file might have been deleted since we started lazygit)

Reply:
what kind of mad lad would remove config files while a program is running, that's like replacing a car component while it's driving but sure we handle this so why not support it.